### PR TITLE
Fix tokenization on TEDLIUM2 in IWSLT21 ASR recipe

### PR DESCRIPTION
**Pull request by hirofumi0810**
Added tokenization with moses for TEDLIUM2 in `stage-0`.

### CODE REVIEW

**Comment by hirofumi0810** [comment]
@sw005320 @ftshijt This PR is ready.

File: egs/iwslt21/asr1/run.sh [hunk 1]
```diff
@@ -126,7 +126,7 @@ if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
     # additionally we copy text to text.${case}
     for x in tr_${data_code} dt_${data_code} et_${data_code}_dev et_${data_code}_test; do
         for case in tc lc lc.rm; do
-            cp data/${x}/text data/${x}/text.${case}
+            paste -d " " <(cut -d " " -f 1 data/${x}/text) <(cut -d " " -f 2- data/${x}/text | lowercase.perl | tokenizer.perl -l en -q) > data/${x}/text.${case}
```
Comment (ftshijt) [inline egs/iwslt21/asr1/run.sh:129]: it seems that we also do the `lowercase` for case `tc`, is it intentional?

Comment (ftshijt) [reply egs/iwslt21/asr1/run.sh:129]: I understand it's for asr training, so this might not be that important though.

Comment (hirofumi0810) [reply egs/iwslt21/asr1/run.sh:129]: @ftshijt Because TEDLIUM2 is already lowercased, I think it's fine.

Comment (ftshijt) [reply egs/iwslt21/asr1/run.sh:129]: Cool~

**Comment by ftshijt** [comment]
Merged! Thanks for the update.

**Pull request merged**